mailpoet/eslint.config.js

```diff
@@ -26,4 +26,27 @@ module.exports = [
     ...config,
     files: ['tests/javascript_newsletter_editor/**/*.js'],
   })),
+
+  // ES6 config overrides
+  {
+    files: ['assets/js/src/**/*.jsx', 'tests/javascript/**/*.js'],
```

Comment:
I see we also have a couple of .jsx files in the premium plugin also in the assets folder. So I guess these rules will be applied to those as well. Correct?

Reply:
No, this config is not loaded in the premium plugin. In there it's handled by this one: https://github.com/mailpoet/mailpoet-premium/blob/22ba8630b34579ac782175173c1a434c9ff9bc53/eslint.config.js